maestrowf/datastructures/yamlspecification.py

```diff
@@ -41,6 +41,42 @@
 
 logger = logging.getLogger(__name__)
 
+STUDY_STEP_KEYS = {
+    "name": None,
+    "description": None,
+    "run": {
+        "cmd": None,
+        "restart": None,
+        "depends": None,
+        "nodes": None,
+        "procs": None,
+        "cores per task": None,
+        "gpus per task": None,
+        "walltime": None,
+        "num resource set": None,
```

Comment:
I don't think this key is ever used in Maestro.

Reply:
Fixed